### PR TITLE
Set CI build item to error if .shoov.yml is missing

### PR DESCRIPTION
**Pull request by HelenaEksler**
#51

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu Could you please review this PR? (screenshot is in the issue - https://github.com/shoov/shoov-nodejs/issues/51#issuecomment-147358016)

File: routes/ci.js [hunk 1]
```diff
@@ -63,7 +63,8 @@ module.exports = function(config, logger) {
       // Get Shoov configuration file from repository.
       .then(function(response) {
         if (!response) {
-          throw new Error("Can't get .shoov.yml");
+          // Send other type of an error if file doesn't exist at all.
+          throw new ReferenceError("Can't get .shoov.yml");
```
Comment (amitaibu) [inline routes/ci.js]: Lets extend Error with our own Error (e.g. `BadRequestError`).

File: routes/ci.js [hunk 2]
```diff
@@ -11,6 +11,18 @@ var debug = false;
 var conf = {};
 var log = {};
 
+/**
+ * Implements custom Error type "FileNotFoundError". Extends Error.
```
Comment (HelenaEksler) [inline routes/ci.js]: @amitaibu Not sure about Docs for this.

Comment (amitaibu) [inline routes/ci.js]: Maybe `Extends error object to indicate a missing .shoov.yml file.`

File: routes/ci.js [hunk 3]
```diff
@@ -11,6 +11,20 @@ var debug = false;
...
+/**
+ * Implements custom error type "FileNotFoundError".
+ * Extends Error object and indicates an error regarding the .shoov.yml file
+ * is missing.
+ *
+ * @param message
+ *  Error message
+ */
+function FileNotFoundError(message) {
+  this.name = "FileNotFoundError";
```
Comment (amitaibu) [inline routes/ci.js]: " => '

File: routes/ci.js [hunk 4]
```diff
@@ -11,6 +11,20 @@ var debug = false;
 var conf = {};
 var log = {};
 
+/**
+ * Implements custom error type "FileNotFoundError".
+ * Extends Error object and indicates an error regarding the .shoov.yml file
```
Comment (amitaibu) [inline routes/ci.js]: line break.

**Comment by HelenaEksler** [comment]
@amitaibu Ready.

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**